@ui/ui-kit/Icon/MakeIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react'
+import { MakeIconPropsType } from './iconsType'
+import style from './IconsComponent.module.scss'
+import Link from 'next/link'
+
+export const MakeIcon: FC<MakeIconPropsType> = (props) => {
```

Comment:
you can destructure values just inside props:

export const MakeIcon: FC<MakeIconPropsType> = ({ OutlineIcon, Icon, className, text, isActive, isDisabled, url }) => {
}

Reply:
This is a good point, but in my opinion it makes the function declaration way too long. So I moved it to the next line

like this: 

export const MakeIcon: FC<MakeIconPropsType> = (props) => {
  const { OutlineIcon, Icon, className, text, isActive, isDisabled, url } = props
}